package.json

```diff
@@ -3,6 +3,9 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@emotion/react": "^11.8.2",
+    "@emotion/styled": "^11.8.1",
+    "@mui/material": "^5.5.1",
```

Comment:
You only use MUI for cards which can be handled easily without MUI.
I'm afraid this will increase bundle size.

Reply:
Okay